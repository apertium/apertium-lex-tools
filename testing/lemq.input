^a<n><cmp>/b<n>/c# d<n>$
^e<n><cmp>/f<n>/g<n>$
