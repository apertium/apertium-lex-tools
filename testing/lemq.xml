<rules>
  <rule weight="2.0">
    <match tags="*.cmp">
      <remove contains="#"/>
    </match>
  </rule>
</rules>
