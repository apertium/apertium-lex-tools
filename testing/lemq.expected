^a<n><cmp>/b<n>$
^e<n><cmp>/f<n>/g<n>$
